tests/bootstrap_project/test/module_test.dart

```diff
@@ -11,30 +11,16 @@ const tempDirName = 'temp';
 
 void main() {
   final rootPath = path.join(Directory.current.path, '..', '..');
-  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
+  final cliProjectPath = getServerpodCliProjectPath(rootPath: rootPath);
+  final cliDartEntrypoint = getServerpodCliEntrypointPath(rootPath: rootPath);
   final tempPath = path.join(rootPath, tempDirName);
 
   setUpAll(() async {
-    await runProcess(
-      'dart',
-      ['pub', 'global', 'activate', '-s', 'path', '.'],
-      workingDirectory: cliPath,
-    );
```

Comment:
I decided to migrate our bootstrap tests to use `dart run` as pub caching was causing a lot of issues while making changes to the CLI. And since this is not a new pattern, where our scripts in `./util` also use `dart run` directly, it's not an uncommon pattern.

It does increase the time to run the tests as each `dart run` will take time to start up, but in the future with the introduction of `dart install` in 3.10, we can migrate to that which does not have this issue as far as I know.

If the increase in time is very undesired, it would be trivial to compile the CLI ourselves via `dart compile exe` in `setUpAll` to mitigate slightly the delay